lib/fog/azurerm.rb

```diff
@@ -75,6 +75,10 @@ module AzureRM
 
   # Main AzureRM fog Provider Module
   module AzureRM
+    # We cannot necessarily set the DEBUG environment variable for fog-azure-rm because
+    # that would log the complete HTTP body since this variable is also used by the Azure storage SDK.
+    # Instead let's offer another way to enable fog-azure-rm debug logging
+    Fog::Logger.instance_variable_get(:@channels)[:debug] = ::STDERR if ENV['FOG_DEBUG']
```

Comment:
Hey, could you tweak the syntax here to use the existing helpers? I think you should be able to just do this: `Fog::Logger[:debug] = ::STDERR`. Otherwise, by reaching directly into the instance variables it may increase the likelihood of accidentally breaking this in the future, vs using the public interface, which should hopefully remain stable.